.github/ISSUE_TEMPLATE/010-bug-compilation.yml

```diff
@@ -65,6 +65,15 @@ body:
         If possible, please do a git bisect and identify the exact commit that introduced the bug.
     validations:
       required: false
+  - type: textarea
+    id: command
+    attributes:
+      label: Compile command
+      description: >
+        Please provide the exact command you used to compile llama.cpp. For example: `cmake -B ...`.
+      render: shell
+    validations:
+      required: true
```

Comment:
If we have a dedicated field for the command I think the instruction that the logs should contain the command can be removed.